Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -791,22 +803,46 @@ final class StatusTableViewController: LoopChartsTableViewController {
     // MARK: - Table view data source
 
     override func numberOfSections(in tableView: UITableView) -> Int {
-        return Section.count
+        return Section.allCases.count
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         switch Section(rawValue: section)! {
+        case .tempStatus:
+            return notificationsCriticalAlertPermissionsViewModel.showWarning ? 1 : 0
         case .hud:
             return shouldShowHUD ? 1 : 0
         case .charts:
-            return ChartRow.count
+            return ChartRow.allCases.count
         case .status:
-            return shouldShowStatus ? StatusRow.count : 0
+            return shouldShowStatus ? StatusRow.allCases.count : 0
         }
     }
 
+    private class TempStatusCell: UITableViewCell {
```

Comment:
[nit] This is a very specific type of cell for "Alert Permissions Disabled", but given the generic and not-very-descriptive name of "TempStatusCell". Consider a more explicit name. If it ever becomes more generic and holds a variety of information we can change then.

Reply:
Good point.  I picked `AlertPermissionsDisabledWarningCell`. Hope that is clearer.